tools/schemacode/bidsschematools/__main__.py

```diff
@@ -1,36 +1,17 @@
-import logging
-import os
+import argparse
 
-import click
+import argh
 
-from .schema import export_schema, load_schema
+from .schema import export
+from .validator import validate_bids
 
 
-@click.group()
-@click.option("-v", "--verbose", count=True)
-def cli(verbose):
-    """BIDS Schema Tools"""
-    logging.getLogger("bidsschematools").setLevel(logging.INFO - verbose * 10)
```

Comment:
We've lost a way to debug. Does `argh` have anything like this?

Reply:
not entirely sure what this is supposed to do. However, `argh` is fully compatible with the vanilla `argparse` parsers, so we can add whatever additional arguments we want. I guess a global argument wouldn't contravene to the automatic CLI generation concept... so sure, why not.

Problem is I'm not quite sure how to test that this feature is working properly. The argparse API is much like click, e.g.:

```
parser.add_argument('-v', '--verbose', action='store_true')
```

could you add a test for this or share some example behaviour I could try to reproduce?

Reply:
@effigies does [e483420](https://github.com/bids-standard/bids-specification/pull/1415/commits/e483420b5b4056d79241b432158b165a7a18b06b) address this?